news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+
+At the PrestaShop Quality Assurance team (aka QA), we browse issues on GitHub all day long. As we go about doing that, we often come across messages from people who are angry about a bug taking a long time to be fixed, and saying "this is a major issue, why is it still not fixed?”, or “This bug is marked as minor but it should be major”. Everyone agrees that not all bugs can be fixed at once, and that the more critical bugs should be fixed first. The problem is that since this appreciation is subjective, any opinion will almost always be challenged by the people who are personally affected by that specific issue, and who want to get it resolved first.
+
+In order to avoid frustration and misunderstandings, it was time to come up with a better, more formal and transparent definition for the criteria used to qualify issues’ severity, and how they should be applied. 
+
+The first thing to learn is that severity is to be distinguished from priority. Indeed, severity is used to measure the negative impact that a bug has on a system, a feature, a component or on the project development. It is usually defined by the QA team. As for the priority, it is used to organize all the tasks (bugs, improvements, features, technical tasks) that have to be done in order to meet the project’s deadlines. 
+
+
+## Issue Severity Criteria
+
+When a new issue is created, the first step is to understand what the problem is and then reproduce it. Once that work is done, the second step is to define the severity of that bug.
+
+Four severity levels have been defined, as explained below.
+
+
+### Critical
+
+The bug affects critical functionality or critical data and there is no workaround (no way to avoid it).
+A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
+Examples:
+
+- Difficulty accessing the front office or back office (significant slowdown, error during installation or update, fatal error)
+- Difficulty to globally manage categories, products or customers
+- Difficulty to globally place and manage orders
+
+A critical issue should result in a patch version that should be released as soon as possible. Our recent [spam attack](http://build.prestashop.com/news/fighting-against-spamming-again/) is a good example: it was a critical security issue as our users were flooded with fake customers’ accounts. As a consequence, a patch version has been released that resolve it on 1.6 and 1.7, a good example of a community work.
```

Comment:
Are you sure you want to use that example ? Not a security patch instead ? 😄

Reply:
Why not this one ? I think it's a good recent example for which we reacted quickly with patch versions

Reply:
Because I dont agree it was a critical security issue 😄 it was very annoying, it would pollute merchant customer databases, but this was no critical issue. Not the same kind as the vulnerabilities that required the releases of 1.7.3.4 and 1.7.4.4

And also we did not react quickly 😐 we both know why but that was not the best example of quick reaction we have

Reply:
Maybe we can just remove the example?

Reply:
I just found another example with 1.7.2.5, wdyt ?